src/main.cpp

```diff
@@ -2552,6 +2553,8 @@ bool DisconnectBlock(CBlock& block, CValidationState& state, CBlockIndex* pindex
             }
         }
 
+        // a bwt request carries no events to revert, nothing to be done for tx.GetVBwtRequestOut() 
```

Comment:
cancel sidechain events for MCBTR needed to update sc maturing balance

Reply:
Done in mbtr_introduction_final_fixes